<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSMView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tblLog"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartLogging">
        <property name="text">
         <string>Start logging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopLogging">
        <property name="text">
         <string>Stop logging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetLvl1">
        <property name="text">
         <string>Set lvl 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetLvl2">
        <property name="text">
         <string>Set lvl 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetLvl3">
        <property name="text">
         <string>Set lvl 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBlockFile">
        <property name="text">
         <string>Block file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUnblockFile">
        <property name="text">
         <string>Unblock file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAllowUnload">
        <property name="text">
         <string>Allow unload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBlockUnload">
        <property name="text">
         <string>Block unload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFilter">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkX86">
        <property name="text">
         <string>X86</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDmesg">
        <property name="text">
         <string>DMESG</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
